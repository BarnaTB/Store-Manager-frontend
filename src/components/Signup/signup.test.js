import React from 'react';
import { shallow } from 'enzyme';

import SignUp from './signup';

describe('App', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<SignUp />);

    expect(wrapper).toMatchSnapshot();
  });
});
