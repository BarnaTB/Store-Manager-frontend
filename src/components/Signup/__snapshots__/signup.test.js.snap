// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
<div
  className=""
>
  <a
    href="/"
  >
    <h1>
      Store Manager
    </h1>
  </a>
  <p>
    Welcome to your store!
  </p>
  <div
    className=""
  >
    <form
      className="email-form"
      onSubmit={[Function]}
    >
      <input
        id="new-email"
        name="email"
        onChange={[Function]}
        placeholder="Email . . ."
        type="email"
      />
      <br />
      <br />
      <input
        id="new-username"
        name="username"
        onChange={[Function]}
        placeholder="Username . . ."
        type="text"
      />
      <br />
      <br />
      <input
        id="new-password"
        name="password"
        onChange={[Function]}
        placeholder="Password . . ."
        type="password"
      />
      <br />
      <br />
      <input
        id="submit-btn"
        type="submit"
        value="Register"
      />
      <br />
      <br />
    </form>
  </div>
</div>
`;
